### ISSUE TITLE
Azure: Is it possible to use certificate as client credential?

### ISSUE THREAD

**Issue by kasbst** [vouch/vouch-proxy#449]
We are not allowed to use Azure SPN secret credential due to internal company policies. We can use only Azure SPN certificate credential.

Currently, it seems that vouch-proxy only suppport secret credential for Azure oauth2 provider.

Is there any plan / chance that certificate credential support will be implemented?

Thank you!

**Comment by bnfinet** [comment]
@kasbst yes that's correct, VP does not currently support Azure SPN certificate credential.  I'll mark it here as a feature request.
